### DISCUSSION TITLE
Running demo

### DISCUSSION THREAD

**Discussion by praveenray** [mkpaz/atlantafx#51] [General]
Hello
How do I run the demo app? I tried running the sampler from IntelliJ. It runs - however no theme is loaded. Little lost as why it comes up with no theme.  Same result after running the .exe (that I got from running mvnw install). Any pointers would be helpful. Thanks.

**Comment by devrj8** [comment]
Hi @praveenray ,

I followed the documentation. it is pretty clear.  [documentation link](https://mkpaz.github.io/atlantafx/build/#:~:text=theme%20sources%20(SASS)-,Instructions,-To%20build%20and)
`mvn install`
`mvn javafx:run -pl sampler`

The Result looks like below
![image](https://github.com/mkpaz/atlantafx/assets/25859158/be86be1f-176c-4090-90d6-279d772534e1)
